models/order.js

```diff
@@ -21,13 +24,10 @@ OrderSchema.methods.sendSmsNotification = function(message, statusCallback, call
     statusCallback: statusCallback,
   };
 
-  client.messages.create(options)
-    .then((message) => console.log('Message sent to ' + message.customerPhoneNumber))
-    .catch((error) => console.log(error));
-
-  if (callback) {
-    callback.call(self);
-  }
+  return client.messages.create(options)
+    .then((message) => {
+      console.log('Message sent to ' + message.to);
+    })
```

Comment:
Did you consider adding a `catch` method to deal with rejected cases? Moreover, seems like you missed a semicolon. 😉